knpu/install.md

```diff
@@ -137,6 +137,12 @@ For me that's
 java -jar ~/Downloads/selenium-server-standalone-2.45.0.jar
 ```
 
+> Firefox `47.0.0` and fewer is not supported at all since Selenium `3.0.0` - update Firefox
+> to the latest version in order to use the latest Selenium server. Also you need to install
+> the new [geckodriver][geckodriver], but keep in mind that it's based on the evolving W3C
+> WebDriver spec, and uses the wire protocol in that spec, which is liable to change without
```

Comment:
You lost me with everything after "but keep in mind". What should I watch out for and why exactly? Keep in mind, our users basically just want firefox to open and click around - they don't really care how it works (unless it's still buggy). And if chrome is better, we could add a note about using it instead (and how, I can't remember if we talk about using a different browser or not.